compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/SymbolEnter.java

```diff
@@ -2334,7 +2335,7 @@ boolean checkTypeAndVarCountConsistency(BLangTupleVariable varNode, BTupleType t
                             return false;
                         }
                         dlog.error(varNode.pos, DiagnosticErrorCode.INVALID_LIST_BINDING_PATTERN_DECL,
-                                   varNode.getBType());
+                                   bType);
```

Comment:
Shouldn't error messages have the type reference?

Reply:
My bad, fixed in 5b92537. Thanks!